src/api/refreshAccessToken.ts

```diff
@@ -77,6 +81,11 @@ export const refreshAccessTokenWithRetriesOnError = async (
     if (error instanceof ApiError && error.statusCode === 401) {
       return { error: REFRESH_TOKEN_IS_EXPIRED_ERROR }
     }
+    let isConnected
+    await fetchNetInfo().then((state) => {
+      isConnected = state.isConnected
+    })
```

Comment:
pourquoi mélanger `await` avec `.then()` ?

```suggestion
    const { isConnected } = await fetchNetInfo()
```